azerothcore/azerothcore-wotlk
# Fixed join multiple guilds exploit

OLD PR: https://github.com/azerothcore/azerothcore-wotlk/pull/1789

Tested And It works fine for me

##### HOW TO TEST THE CHANGES:

First: Clone WowAddin (WoW Console) from https://github.com/tomrus88/WowAddin
Then: Use below code as a method and register the command in order to function

```
bool GuildLeave(char const*, char const* args)
{
  CDataStore Packet(CMSG_GUILD_DECLINE);
  Packet.Finalize();
  ClientServices::SendPacket(&Packet);

  Console::Write("Packet Sent", ECHO_COLOR);
  return true;
}
```

After That: Try to compile and Inject it to Wow.exe Process using a DLL Injector

Then see this video without this PR: https://vimeo.com/594412340
